packages/core/package.json

```diff
@@ -3,8 +3,16 @@
   "version": "0.42.1",
   "description": "",
   "main": "dist/umd/index.js",
+  "module": "src/index.ts",
   "types": "dist/esm/index.d.ts",
```

Comment:
I'm not sure why you removed the types line - it was the primary issue you were seeing with streaming loader - both the one here and the one in the exports is needed.

Reply:
it is useless when you have `exports`

Reply:
Doesn't build without it